pkg/cmd/image/list.go

```diff
@@ -46,7 +45,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-func List(ctx context.Context, options types.ImageListCommandOptions) error {
+func List(ctx context.Context, options types.ImageListCommandOptions, stdout io.Writer) error {
```

Comment:
```suggestion
func List(ctx context.Context, stdout io.Writer, options types.ImageListCommandOptions) error {
```

Reply:
@AkihiroSuda Shall we somehow (i.e., open an issue?) standardize the parameter order?

- https://github.com/containerd/nerdctl/blob/35bb03ee856dd48c35171ff1c25d80d9239eb35b/pkg/cmd/image/inspect.go#L35
- https://github.com/containerd/nerdctl/pull/1837#discussion_r1072461623

Thanks!

Reply:
Yes
- https://github.com/containerd/nerdctl/issues/1889

Reply:
thanks, fixed. Shall I also update `printImages` from

```go
func printImages(ctx context.Context, client *containerd.Client, imageList []images.Image, options types.ImageListCommandOptions, w io.Writer) error {
```

to

```go
func printImages(ctx context.Context, w io.Writer, client *containerd.Client, imageList []images.Image, options types.ImageListCommandOptions) error {
```

Reply:
I'd prefer to have `client` before `w`, but not a strong opinion.

@Zheaoli WDYT?

Reply:
Updated according to the summary in https://github.com/containerd/nerdctl/pull/1881#discussion_r1083312219, thanks!